### PR TITLE
Add io.thenexusavenger.Nexus-LU-Launcher

### PR DESCRIPTION
**Pull request by TheNexusAvenger**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
  - `--filesystem=home:ro` - Users are required to provide an archive (`zip` or `rar`, typically) of LEGO Universe. Nexus LU Launcher needs to be able to read these files, which are typically going to be in the user's home directory. Write access is not used.
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: io.thenexusavenger.Nexus-LU-Launcher.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - org.freedesktop.Platform.ffmpeg_full.i386
+  - org.winehq.Wine.gecko
+  - org.winehq.Wine.mono
+  - org.winehq.Wine.DLLs
+
+build-options:
+  append-path: /usr/lib/sdk/dotnet8/bin
+  append-ld-library-path: /usr/lib/sdk/dotnet8/lib
+  env:
+    PKG_CONFIG_PATH: /app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet8/lib/pkgconfig
```
Comment (bbhtt) [inline io.thenexusavenger.Nexus-LU-Launcher.yml:36]:
```suggestion
  append-ld-library-path: /usr/lib/sdk/dotnet8/lib
  append-pkg-config-path: /usr/lib/sdk/dotnet8/lib/pkgconfig
```

Comment (TheNexusAvenger) [reply io.thenexusavenger.Nexus-LU-Launcher.yml]: Change made. No issues locally with this change.

**Comment by bbhtt** [comment]
bot, build io.thenexusavenger.Nexus-LU-Launcher

**Comment by bbhtt** [comment]
WineHQ is x86_64 only https://docs.flathub.org/docs/for-app-authors/maintenance#only-build-on-x86_64

**Comment by TheNexusAvenger** [comment]
bot, build io.thenexusavenger.Nexus-LU-Launcher

**Comment by TheNexusAvenger** [comment]
bot, build io.thenexusavenger.Nexus-LU-Launcher

**Comment by TheNexusAvenger** [comment]
Tested on a Ubuntu VM with no issue, but didn't work on a spare laptop because of Wayland. Turns out Avalonia (UI library) does not support Wayland yet. Changing to use X11 worked.

**Comment by TheNexusAvenger** [comment]
Launcher works on my laptop (NixOS 23.11, KDE Plasma 5.27 with Wayland) with the latest change, as well as my Ubuntu 22.04 VM that was already working before.
image

LEGO Universe also launches as well [insert full-screen image of LEGO Universe's login window here], but only with 1 of the 2 archive files people would find online. I can reproduce it on my development system running Windows, so it isn't specific to Linux or Flatpak.

**Comment by barthalion** [comment]
bot, build io.thenexusavenger.Nexus-LU-Launcher

**Pull request closed without merging**